resolve_stream.py

```diff
@@ -147,6 +261,57 @@ def resolve_stream(refdes, method, stream, stream_map, m2m):
                 print '%sNOT FOUND: needed_stream=%r poss_params=%r' % (indent, needed_stream, poss_params)
 
 
+# TODO - need to configure this to either use asset management directly or M2M
```

Comment:
should the following global variables be moved up closer to the top?

Reply:
I'm re-working this part of the code - it will change in the next commit.

Reply:
will change in the next commit